.github/workflows/mcad-release.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow will create a release branch and GitHub release tag
+
+name: mcad Release
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Release tag'
+        required: true
+        default: 'v0.0.0-dev'
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: checkout code
+      uses: actions/checkout@v3
+
+    - name: Configure Git
+      run: |
+        git config user.name "GitHub Actions"
+        git config user.email "github-actions@github.com"
+
+    - name: set release branch
+      id: release-branch
+      run: |
+        if [[ "${{ github.event.inputs.tag }}" =~ "^v\d+\.\d+\.\d+$" ]]; then
+          echo ":: set-output name=release_branch::release-${{ github.events.inputs.tag }}"
+        else
+          echo "Invalid tag format: ${{ github.event.inputs.tag }}"
+          exit 1
+        fi
+
+    - name: Create release branch
```

Comment:
I don't think that release branch is needed to be created as part of release process, it can be created later if there is a need for some backport. See https://github.com/project-codeflare/instascale/pull/75#issuecomment-1580885413

Reply:
Yes agree.. but looking at the **Makefile** configuration which expects branch name starts with **release-** for the mcad image build and push to the release tag **release-v**

Reply:
I think we can refactor this **Makefile**,  when tag got created (eg v1.31.0) build and push the image to release tag